src/markdown-pages/docs/cli-command-reference/runs-get.md

```diff
@@ -3,30 +3,146 @@ path: "/docs/cli-command-reference/cli-runs-get"
 title: "Viewing test run status"
 ---
 
+You can specify a number of parameters on the `galasactl runs get` command to query test runs on specific
+details, and to display the output of those queries in different formats. The following tables provide a high-level summary of a subset of available options. You can view the full command syntax, including parameter descriptions, in the <a href="https://github.com/galasa-dev/cli/blob/main/docs/generated/galasactl_runs_get.md" target="_blank">Galasa cli repository</a>.
 
-Use the test run name to query test run details by using the following command:
 
+Table 1: options that you can set on the `galasactl runs get` command to query test run data on specific details:
+
+| Name |  Description  |
+| :---- | :-------- | 
+| `--name`  | Use the `--name` parameter to query details about completed test runs or tests that are running by using the name of the test run.  |
+| `--requestor`| Use the `--requestor` parameter to view tests results of test runs that were submitted to the ecosystem by a specified person. |
+| `--age`| Use the `--age` parameter to specify a time period in which the tests ran. The _age_ parameter is specified in the format _FROM:TO_. Units of time can be specified in units of weeks _w_, days _d_, or hours _h_. The _TO_ part of the value is optional, with a default set to 0 which indicates the current time. The _FROM_ part of the value specifies how far back in time the query is applied. The _TO_ value must therefore always be a larger value than the _FROM_ value. If the `--name` parameter is specified, the `--age` parameter is not used. |
+
+
+Table 2: options that you can set on the `galasactl runs get` command to display test run results in different formats:
+
+| Name |  Description  |
+| :---- | :-------- | 
+| `--format summary` | The default format is _summary_. The _summary_ format is useful if you just need to know the high-level status and result of a test.  | 
+| `--format details` | The _details_ format is useful when you need to see all the details of a test run, including method information.  | 
+| `--format raw` | The _raw_ format output is useful if you are writing scripts to report on multiple test runs programmatically. The output from `galasactl runs get` is returned in a form that makes it easy for scripting to digest the individual pieces of data available, without having to query the data via the published REST interface. | 
+
```

Comment:
we still query the data with the REST interface. the raw format is just so we don't have headers and nicely fomratted outputs. just the values so it is easier for scripting. 
I think this can be removed: 'without having to query the data via the published REST interface'